pkgs/development/libraries/azure-sdk-for-cpp/core-amqp.nix

```diff
@@ -0,0 +1,78 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  core,
+  c-shared-utility,
+  macro-utils-c,
+  umock-c,
+  nix-update-script,
+  meta,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "azure-sdk-for-cpp-core-amqp";
+  version = "1.0.0-beta.11";
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "Azure";
+    repo = "azure-sdk-for-cpp";
+    tag = "azure-core-amqp_1.0.0-beta.11";
+    hash = "sha256-MQsz5Dmv1BwfUaN1VXMC3hPdMHihlgOBaukp5wgTNJc=";
+  };
+  sourceRoot = "source/sdk/core/azure-core-amqp";
+
+  postPatch = ''
+    sed -i '/CMAKE_CXX_STANDARD/d' CMakeLists.txt
+  '';
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+  ];
+
+  propagatedBuildInputs = [
+    core
+    c-shared-utility
+    macro-utils-c
+    umock-c
+  ];
+
+  env = {
+    AZURE_SDK_DISABLE_AUTO_VCPKG = 1;
```

Comment:
(Not a blocker.)

Is the opposite `AZURE_SDK_DISABLE_AUTO_VCPKG=0` or `AZURE_SDK_DISABLE_AUTO_VCPKG=`? If the latter, consider

```nix
{
  env = {
    AZURE_SDK_DISABLE_AUTO_VCPKG = true;
  };
}
```